src/server/plugins/engine/components/NumberField.ts

```diff
@@ -111,3 +117,31 @@ export class NumberField extends FormComponent {
     return typeof value === 'number'
   }
 }
+
+export function getValidatorPrecision(component: NumberField) {
```

Comment:
Prob being pedantic here (sorry!), but this func is returning a validator so maybe `getPrecisionValidator` is more apt? Initially reading this I thought the function returns a precision value.

Reply:
Yeah it makes sense now 😆

We have `getValidatorXXX` elsewhere but if you fancy changing things like this in another PR?

E.g. I'm slowly tackling confusing names in this one for example:

* https://github.com/DEFRA/forms-runner/pull/515